integration/spark/src/main/scala/org/apache/carbondata/indexserver/DistributedPruneRDD.scala

```diff
@@ -143,6 +143,7 @@ private[indexserver] class DistributedPruneRDD(@transient private val ss: SparkS
         val reader = dataMapFormat.createRecordReader(inputSplit, attemptContextImpl)
         reader.initialize(inputSplit, attemptContextImpl)
         while (reader.nextKeyValue()) {
+          reader.getCurrentValue.setRowCount(reader.getCurrentValue.getDetailInfo.getRowCount)
```

Comment:
getCurrentValue is called 3 times here now !

better to call once and set value or handle it inside.